common/app/views/fragments/amp/stylesheets/ads.scala.html

```diff
@@ -1,5 +1,8 @@
 .amp-ad-container {
     background: #f6f6f6;
+    background-image: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IiB2aWV3Qm94PSIwIDAgODYgNzYiIGVuYWJsZS1iYWNrZ3JvdW5kPSJuZXcgMCAwIDg2IDc2IiB4bWw6c3BhY2U9InByZXNlcnZlIj48cGF0aCBmaWxsPSIjZGNkY2RjIiBkPSJNNTAuOCwxNS45YzAtMTEuOC0zLjUtMTMuNi03LjgtMTMuNmMtNC4yLDAtNy43LDEuNi03LjcsMTMuNmMwLDEyLDMuNSwxMyw3LjcsMTNDNDcuMiwyOC45LDUwLjgsMjcuNyw1MC44LDE1LjkgTTM1LjMsNjAuOGMtNS40LDAtNyw0LTcsNi45YzAsNC4zLDMuOCw4LjIsMTUsOC4yYzEyLjcsMCwxNi4yLTMuNiwxNi4yLTguMmMwLTQuMS0zLjEtNi45LTguMi02LjlIMzUuM3ogTTI1LjEsMC41QzEwLjMsNy4zLDAsMjIuMywwLDM5LjZDMCw1MS40LDQuNyw2MiwxMi40LDY5LjhjMC0wLjIsMC0wLjQsMC0wLjZjMC03LjcsNy41LTEwLjUsMTQuMS0xMS45djBjLTYuMy0xLjQtOS40LTUuOS05LjQtMTAuNWMwLTYuMiw2LjktMTEuNSwxMC4zLTEzLjlMMjcsMzIuN2MtNS45LTMuMy05LjgtOS05LjgtMTYuOEMxNy4yLDkuMiwyMC4xLDQsMjUuMSwwLjUgTTg2LDM5LjZDODYsMjEuOSw3NS4zLDYuNyw2MCwwLjFjNS4xLDMuMyw4LjMsOC41LDguNSwxNWMwLDAuNiwwLjEsMSwwLjEsMS40YzAsMTIuOC0xMC40LDE5LjYtMjUuNiwxOS42Yy0zLjksMC02LjUtMC4yLTkuNy0xLjJjLTEuNSwxLTIuNiwyLjctMi42LDQuM2MwLDIuMSwxLjksMy44LDQuMiwzLjhoMjEuMWMxMy4xLDAsMTkuNSw1LjQsMTkuNSwxNi45YzAsMy45LTAuOCw3LjMtMi40LDEwLjNDODEsNjIuNSw4Niw1MS42LDg2LDM5LjYiLz48L3N2Zz4=);background-size: 105px;
```

Comment:
Not sure which is the best solution :confused: . I will look into it before adding any more things like this to the AMP pages